tools/check_metadata

```diff
@@ -4,6 +4,7 @@ success=1
 for file in $files; do
     grep -q '# Summary: .\+' $file || (echo "Missing '# Summary: <multi line summary of test>'" in $file && exit 1) || success=0
     grep -q '# Maintainer: .\+@.\+' $file || (echo "Missing '# Maintainer: <email address>'" in $file && exit 1) || success=0
+    grep -q '# Copyright .\+' $file || (echo "Missing Copyright © <year> SUSE LLC" in $file && exit 1) || success=0
```

Comment:
is within the scope of this pr to also check if one of the files being modified, has an updated header? otherwise lgtm

Reply:
no, not yet, it will be handled in another pr (when I am back from vacation :) )